src/notation/view/widgets/editstyle.ui

```diff
@@ -8293,7 +8293,7 @@
                  <item>
                   <widget class="QLabel" name="label_62">
                    <property name="text">
-                    <string>Position:</string>
+                    <string>Placement:</string>
```

Comment:
Is it intentional that these are all changed back to Placement? Not too long ago, we intentionally changed them to Position, so I was somewhat surprised to see them changed back while reviewing last week's strings changes.

Reply:
Yes, there is a new "Position" property which controls the position of text's bounding box relative to its reference separately to text alignment. We'll enable this for all text eventually, but for now it's just chord symbols and marker text - https://github.com/musescore/MuseScore/pull/28377